### ISSUE TITLE
Missing 3.3.11 release

### ISSUE THREAD

**Issue by Nerian** [code-and-effect/effective_datatables#68]
Hi,

I updated effective_datatables to 3.3.11 and my datatables stopped working. I took a look this repo to see the changes but master didn't have those new commits. So I dug in the local gem and I noticed that the last pull request (filters stuff) seem to be missing, and that's why my tables are failing.

3.3.11 is not pushed to master, but the release 3.3.11 is there: https://github.com/code-and-effect/effective_datatables/releases

So if I click on it, we can see the tree:

https://github.com/code-and-effect/effective_datatables/commits/063416d95bfcfc9a463724d81b99daa5cd9dff0b

Which has diverged from master:

https://github.com/code-and-effect/effective_datatables/commits/master

Notice the commits "Version 3.3.10" in one tree and the other are different.

The commits on Jan 16, 2018 are missing on the currently released gem.

I guess you started working on new changes without pulling the latest master. Then hit release but the next version on your local (3.3.10) was already released and failed. So you incremented the version, commited and tried releasing again. The gem pushed, but the code didn't, because it conflicted with master. And that's how we got here :)

**Comment by matt-riemer** [comment]
Ack! Sorry about this :)

That's exactly what happened too.  I hate to push a bad build...I've been changing laptops, and I had some weirdness going on when I pushed that version due to my rubygems username/password being missing.

I think this is all fixed in 3.3.13  (unlucky, woo!)

And I updated my gem_release script so this shouldn't happen again

https://github.com/code-and-effect/effective_developer/blob/master/bin/gem_release
